openwisp_users/admin.py

```diff
@@ -271,6 +275,43 @@ def has_change_permission(self, request, obj=None):
             return False
         return super().has_change_permission(request, obj)
 
+    def get_actions(self, request):
+        actions = super().get_actions(request)
+        if not request.POST.get('post') and 'delete_selected' in actions:
+            del actions['delete_selected']
+        return actions
+
+    def delete_selected_modified(self, request, queryset):
+        if not request.user.is_superuser:
+            org_owners = OrganizationOwner.objects.filter(
+                organization_id__in=request.user.organizations_pk
+            ).values_list('organization_user__user', flat=True)
+            qs = queryset.exclude(pk__in=org_owners)
+            if queryset.count() and not qs.count():
+                self.message_user(
+                    request, _('Can not delete organization owners'), messages.ERROR
+                )
+                redirect_url = reverse(
+                    f'admin:{self.model._meta.app_label}_user_changelist'
+                )
+                return HttpResponseRedirect(redirect_url)
+            count = org_owners.count()
+            if count:
+                self.message_user(
+                    request,
+                    ngettext(
+                        '%d user will not be deleted',
```

Comment:
`user can't be deleted because it's an organization owner: {users}.`

Reuse the same list of user from the previous requested change, if it's only one user there will be only one username without commas.